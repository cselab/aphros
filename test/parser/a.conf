
# asdf adsf 

include b.conf

#  adsfasdf 
set double e   3.1   


## adfa s
