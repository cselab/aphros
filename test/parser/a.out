set string c a b   s c    
set int b 123
set double a 2.9
set double e 3.1
set vect f 1 2 34 5 
